recording.py

```diff
@@ -113,5 +146,12 @@ def find_serial_port():
     return FTDIlist[0].name
 
 
+def get_classifier(classifier_filename):
```

Comment:
good idea having this in a separate function, but maybe `load_classifier` would be more appropriate